app/controllers/subscriber_lists_controller.rb

```diff
@@ -18,37 +16,28 @@ def show
   end
 
   def create
-    list = create_or_fetch_subscriber_list
-
+    list = build_subscriber_list
     if list.save
       respond_to do |format|
         format.json { render json: list.to_json, status: 201 }
       end
     else
       respond_to do |format|
-        format.json { render json: {message: "Couldn't create the subscriber list"}, status: 422 }
+        format.json { render json: {message: list.errors.full_messages.to_sentence}, status: 422 }
       end
     end
   end
 
 private
-  def create_or_fetch_subscriber_list
-    gov_delivery = Services.gov_delivery
-
-    response = gov_delivery.create_topic(params[:title])
-
-    SubscriberList.new(
-      title: params[:title],
-      gov_delivery_id: response.to_param,
-      tags: params[:tags]
+  def build_subscriber_list
+    gov_delivery_response = Services.gov_delivery.create_topic(params[:title])
+    SubscriberList.build_from(
```

Comment:
Does the `build_from` factory method add anything now? We can now use `SubscriberList.new(subscriber_list_params.merge(gov_delivery_id: gov_delivery_response.to_param))`.